#!/usr/bin/env python
PACKAGE = "sibiu_nano_p_control"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("Thruster_1",    double_t,    1, "input thruster 1", 0,  -150.0, 150)
gen.add("Thruster_2",    double_t,    1, "input thruster 2", 0,  -150.0, 150)
gen.add("Thruster_3",    double_t,    1, "input thruster 3", 0,  -150.0, 150)
gen.add("Thruster_4",    double_t,    1, "input thruster 4", 0,  -150.0, 150)
gen.add("Thruster_5",    double_t,    1, "input thruster 5", 0,  -150.0, 150)
gen.add("Thruster_6",    double_t,    1, "input thruster 6", 0,  -150.0, 150)
gen.add("Thruster_7",    double_t,    1, "input thruster 7", 0,  -150.0, 150)
gen.add("Thruster_8",    double_t,    1, "input thruster 8", 0,  -150.0, 150)

exit(gen.generate(PACKAGE, "sibiu_nano_p_control", "Tutorials"))

